onnx/common/path.h

```diff
@@ -8,17 +8,67 @@
 #pragma once
 
 #include <string>
+#ifdef _WIN32
+#include <windows.h>
+#endif
 
 namespace ONNX_NAMESPACE {
 
 #ifdef _WIN32
 const std::string k_preferred_path_separator = "\\";
+const std::wstring w_k_preferred_path_separator = L"\\";
```

Comment:
Please use "constexpr const char*" or "constexpr const wchar_t*" so that they can be initialized at compile time. We should avoid having dynamic initializers whenever possible.

Reply:
And on Windows you can use "path::make_preferred" function